database/comments.js

```diff
@@ -16,15 +16,15 @@ const commentSection = new mongoose.Schema({
 
```

Comment:
I think this is more of an index.js file than a comments.js file. The primary usage of this file seems to be initializing the database schema and models, not very related to comments

Reply:
I have moved over most of comment.js's code into index.js which is the primary file for initializing database and left the helper function in comment.js